addons/api/addon/abilities/target.js

```diff
@@ -34,10 +34,25 @@ export default class TargetAbility extends ModelAbility {
     return this.hasAuthorizedAction('add-credential-sources');
   }
 
+  /**
+   * @type {boolean}
+   */
+  get canAddInjectedApplicationCredentialSources() {
+    return (
+      !this.model.isTCP && this.hasAuthorizedAction('add-credential-sources')
```

Comment:
Can we update this logic to the below suggestion? As of right now, the business rules state only SSH targets can inject credentials. `!this.model.isTCP...` is slightly confusing. It could imply we only want to exclude TCP targets from credential injection.
```suggestion
      this.model.isSSH && this.hasAuthorizedAction('add-credential-sources')
```